app/forms/update_user_profile_form.rb

```diff
@@ -4,7 +4,7 @@ class UpdateUserProfileForm
   include FormMobileValidator
   include FormPasswordValidator
 
-  attr_accessor :mobile, :email, :current_password, :password, :totp_enabled
+  attr_accessor :user, :mobile, :email, :current_password, :password, :totp_enabled
```

Comment:
I thought it was needed so that the user_flash_updater.rb could read the user, but attr_reader is better.  Done.